scripts/globals/magic.lua

```diff
@@ -1034,6 +1034,13 @@ xi.magic.getMagicHitRate = function(caster, target, skillType, element, effect,
         xi.msg.debugValue(caster, "Skillchain Bonus Magic Accuracy", magicacc)
     end
 
+    -- Apply bonus macc from TandemStrike
+    local tandemBonus = xi.magic.handleTandemStrikeBonus(caster)
+    if tandemBonus > 0 then
+        magicacc = magicacc + tandemBonus
+        xi.msg.debugValue(caster, "Tandem Strike Magic Accuracy Bonus", magicacc)
+    end
+
```

Comment:
You can add this inside `local function getSpellBonusAcc(caster, target, spell, params)` with the other maccBonus spells instead of here as well

Reply:
I dont this this will apply in all the cases required.
Currently I see this logical gate on applying getSpellBonusAcc: which limits the application to magical skills.
```
if spell ~= nil and skill >= 32 and skill <= 45 then
      magicaccbonus = getSpellBonusAcc(caster, target, spell, params)
end
```

The magic acc provided should apply to all spells cast by the bst but also apply to:
- elemental weaponskills
- Hybrid weaponskills
- Bst Pet - mob magical ready moves (jugs)
- Bst Pet - mob magical sic moves (charmed mobs)

Note to self - currently automaton pet magic is not included in these checks (skill 24)